TerminalDocs/customize-settings/key-bindings.md

```diff
@@ -0,0 +1,495 @@
+
+**Command name:** `moveFocus`
+
+**Default bindings:**
+
+```json
+{ "command": { "action": "moveFocus", "direction": "down" }, "keys": "alt+down" },
+{ "command": { "action": "moveFocus", "direction": "left" }, "keys": "alt+left" },
+{ "command": { "action": "moveFocus", "direction": "right" }, "keys": "alt+right" },
+{ "command": { "action": "moveFocus", "direction": "up" }, "keys": "alt+up" }
+```
+
+#### Actions
+
+| Name | Necessity | Accepts | Description |
+| ---- | --------- | ------- | ----------- |
+| `direction` | Required | `"left"`, `"right"`, `"up"`, `"down"` | Direction in which the focus will move. |
+
+### Resize a pane
+
+This changes the size of the active pane.
+
+**Command name:** `resizePane`
+
+**Default bindings:**
+
+```json
+{ "command": { "action": "resizePane", "direction": "down" }, "keys": "alt+shift+down" },
+{ "command": { "action": "resizePane", "direction": "left" }, "keys": "alt+shift+left" },
+{ "command": { "action": "resizePane", "direction": "right" }, "keys": "alt+shift+right" },
+{ "command": { "action": "resizePane", "direction": "up" }, "keys": "alt+shift+up" }
+```
+
+#### Actions
+
+| Name | Necessity | Accepts | Description |
+| ---- | --------- | ------- | ----------- |
+| `direction` | Required | `"left"`, `"right"`, `"up"`, `"down"` | Direction in which the pane will be resized. |
+
+### Split a pane
+
+This halves the size of the active pane and opens another. Without any arguments, this will open the default profile in the new pane. If an action is not specified, the default profile's equivalent setting will be used.
+
+**Command name:** `splitPane`
+
+**Default bindings:**
+
+```json
+{ "command": { "action": "splitPane", "split": "auto", "splitMode": "duplicate" }, "keys": "alt+shift+d" },
+{ "command": { "action": "splitPane", "split": "horizontal"}, "keys": "alt+shift+-" },
+{ "command": { "action": "splitPane", "split": "vertical"}, "keys": "alt+shift+plus" }
+```
+
+#### Actions
+
+| Name | Necessity | Accepts | Description |
+| ---- | --------- | ------- | ----------- |
+| `split` | Required | `"vertical"`, `"horizontal"`, `"auto"` | How the pane will split. `"auto"` will split in the direction that provides the most surface area. |
+| `commandLine` | Optional | Executable file name as a string | Executable run within the pane. |
+| `startingDirectory` | Optional | Folder location as a string | Directory in which the pane will open. |
+| `tabTitle` | Optional | String | Title of the tab when the new pane is focused. |
+| `index` | Optional | Integer | Profile that will open based on its position in the dropdown (starting at 0). |
+| `profile` | Optional | Profile's name or GUID as a string | Profile that will open based on its GUID or name. |
+| `splitMode` | Optional | `"duplicate"` | Controls how the pane splits. Only accepts `"duplicate"`, which will duplicate the focused pane's profile into a new pane. |
+
+<br />
+
+___
+
+## Clipboard integration commands
```

Comment:
_somewhere_ we should mention that `ctrl+c` and `ctrl+v` are defined in the settings.json.